.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+
+jobs:
+  install-dependencies:
+    docker:
+      - image: circleci/node:lts
```

Comment:
Installing modules with different node version that building will probably cause trouble, for example with node-sass.